pkg/transaction/wrapped/wrapped.go

```diff
@@ -16,7 +17,16 @@ import (
 	"github.com/ethersphere/bee/v2/pkg/transaction"
 )
 
-var _ transaction.Backend = (*wrappedBackend)(nil)
+const (
+	MinimumGasTipCap  = 1_500_000_000 // 1.5 Gwei
+	PercentageDivisor = 100
+	BaseFeeMultiplier = 2
```

Comment:
I would keep these constants in the `transaction` package as they are general. That would avoid importing a specific implementation of `wrapped` package, like in the test file, which is now dependent on the `wrapped` package.

Reply:
Great, thanks.